crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -3591,28 +3592,21 @@
     "EVAL_FALSE"
   ],
   [
-    "2147483648 0",
-    "ADD NOP",
-    "",
-    "UNKNOWN_ERROR",
-    "arithmetic operands must be in range [-2^31...2^31] "
-  ],
-  [
-    "-2147483648 0",
+    "-9223372036854775808 0",
     "ADD NOP",
     "",
     "UNKNOWN_ERROR",
-    "arithmetic operands must be in range [-2^31...2^31] "
+    ""
   ],
   [
     "2147483647",
     "DUP ADD 4294967294 NUMEQUAL",
     "",
-    "UNKNOWN_ERROR",
-    "NUMEQUAL must be in numeric range"
+    "OK",
+    "NUMEQUAL is in numeric range"
   ],
   [
-    "'abcdef'",
```

Comment:
Can you keep test, and change result and description (and also keep the new one)?

Reply:
commit 955424cd4c1dd72f4c1028b3bcdd3dd017bb6309